include/uapi/sound/sof/abi.h

```diff
@@ -27,7 +27,7 @@
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 3
-#define SOF_ABI_PATCH 0
+#define SOF_ABI_PATCH 1
```

Comment:
@plbossart @bardliao i never really understood what the SOF_ABI_PATCH is meant for. Why do we increment patch here and not MAJOR?

Reply:
@ranj063 Please see https://semver.org/

Reply:
Thanks @bardliao so are we going to drop this PR and just send the offset to the logger instead?

Reply:
@ranj063 Yes. I am going to use debugfs to set/get the time stamp instead of ipc